queue_services/entity-filer/src/entity_filer/filing_meta.py

```diff
@@ -61,6 +61,7 @@ class FilingMeta:
 
     application_date: Optional[datetime] = None
     legal_filings: list = field(default_factory=list)
+    dissolution_type: Optional[str] = None
```

Comment:
this will get added to filings we don't want to have it in.

either add it in the dissolution processor, or have it stripped out in the dict.

Reply:
ok. Done!